### PR TITLE
update manage-compute-resources-container-zh

### PR DESCRIPTION
**Pull request by gaoguangze111**
Update update manage-compute-resources-container-zh
Task https://github.com/k8smeetup/website-tasks/issues/3484

### CODE REVIEW

File: content/zh/docs/concepts/configuration/manage-compute-resources-container.md [hunk 1]
```diff
@@ -55,6 +69,15 @@ through the Kubernetes API server.
 
 *CPU* 和*内存*都是*资源类型*。资源类型具有基本单位。CPU 的单位是核心数，内存的单位是字节。
 
+如果您使用的是Kubernetes v1.14或更高版本，则可以指定巨大页资源。巨大页是Linux特有的功能，节点内核在其中分配的内存块比默认页面大小大得多。
```
Comment (tengqm) [inline content/zh/docs/concepts/configuration/manage-compute-resources-container.md]: hugepage 通常译为“巨页“

File: content/zh/docs/concepts/configuration/manage-compute-resources-container.md [hunk 2]
```diff
@@ -55,6 +69,15 @@ through the Kubernetes API server.
...
+
+例如，在默认页面大小为4KiB的系统上，您可以指定一个限制，`hugepages-2Mi：80Mi`。如果容器尝试分配40多个2MiB大页面（a总共80 MiB），则分配失败。
```
Comment (tengqm) [inline content/zh/docs/concepts/configuration/manage-compute-resources-container.md]:
```suggestion
例如，在默认页面大小为 4KiB 的系统上，您可以指定一个限制，`hugepages-2Mi: 80Mi`。如果容器尝试分配超过 40 个2MiB 大页面（总共 80 MiB ），则分配失败。
```

File: content/zh/docs/concepts/configuration/manage-compute-resources-container.md (hunk 1 above)
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/manage-compute-resources-container.md]: 中英文之间通常需要空格。是 Kubernetes v1.14 或

**Comment by markthink** [comment]
@gaoguangze111  需解决一下冲突问题

**Comment by gaoguangze111** [comment]
@markthink  您好，我会跟进的，最近比较忙，没有改这个，不好意思。这两天我会改完。

File: content/zh/docs/concepts/configuration/manage-compute-resources-container.md [hunk 3]
```diff
@@ -284,7 +307,7 @@ the monitoring system.
 
 Pod 的资源使用情况被报告为 Pod 状态的一部分。
 
-如果为集群配置了 [可选监控](/docs/tasks/debug-application-cluster/resource-metrics-pipeline/)，则可以从监控系统检索 Pod 资源的使用情况。
+如果为集群配置了 [可选监控](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/cluster-monitoring/README.md)，则可以从监控系统检索 Pod 资源的使用情况。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/manage-compute-resources-container.md:310]: @gaoguangze111 
Thanks for your job!
pls deal with the conflicts and squash your commits.

**Comment by tengqm** [comment]
Thanks
/lgtm
/approve

**Pull request merged**